forgevtt-module.js

```diff
@@ -97,6 +97,36 @@ class ForgeVTT {
     static isObjectEmpty(obj) {
         return !obj || typeof(obj) !== "object" || Object.keys(obj).length === 0;
     }
+
+    static hasFoundryGlobal = null;
+    /**
+     * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
+     * @param {string} version The version to check
+     * @param {string} target The version to check against
+     * @returns {boolean} True when the version is newer than the target, false otherwise
+     */
+    static isNewerVersion(version, target) {
+        if (this.hasFoundryGlobal !== null) {
+            if (this.hasFoundryGlobal) {
+                return foundry.utils.isNewerVersion(version, target);
+            } else {
+                return isNewerVersion(version, target);
+            }
+        }
+        // this.hasFoundryGlobal hasn't been set yet, so we need to check if the global foundry object exists
+        try {
+            if (foundry && foundry.utils && foundry.utils.isNewerVersion) {
+                this.hasFoundryGlobal = true;
+                return foundry.utils.isNewerVersion(version, target);
+            } else {
+                return isNewerVersion(version, target);
+            }
+        } catch (err) {
+            this.hasFoundryGlobal = false;
+            return isNewerVersion(version, target);
+        }
+    }
```

Comment:
I don't think the `try/catch` is much more expensive than caching a property, we should be able to simplify?
```suggestion
    /**
     * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
     * @param {string} version The version to check
     * @param {string} target The version to check against
     * @returns {boolean} True when the version is newer than the target, false otherwise
     */
    static isFoundryNewerVersion(version, target) {
        try {
            return foundry.utils.isNewerVersion(version, target);
        } catch (_) {
            return isNewerVersion(version, target);
        }
    }
```

Reply:
I guess I could do some measurements to quantify whether it's worth worrying about. If it isn't too expensive then your version would be so much nicer for clarity and maintenance!